project.clj

```diff
@@ -102,7 +102,7 @@
   :lein-ezbake {:vars {:user "puppet"
                        :group "puppet"
                        :build-type "foss"
-                       :java-args ~(str "-Xms2g -Xmx2g -XX:MaxPermSize=256m "
+                       :java-args ~(str "-Xms2g -Xmx2g "
```

Comment:
Yep, I've done that and dropped references to this setting in the documentation as well.